tools/common_lib/src/gemm.h

```diff
@@ -2533,6 +2563,7 @@ class GemmCmDispatcher : public GemmBaseDispatcher
         }
 
 private:
+    bool use_stateless_;
```

Comment:
this should be part of `cm_params_t` (as I've already stated in another comment)

Reply:
done